tests/tracker/index_test.py

```diff
@@ -82,83 +67,36 @@ def test_extract_index_create(client):
-                    "location": {
-                        "type": "geo_point"
-                    }
-                }
-            },
+            "mappings": {"dynamic": "strict", "properties": {"location": {"type": "geo_point"}}},
             "settings": {
                 "index": {
                     "number_of_shards": "3",
                     "provided_name": "osmgeopoints",
                     "creation_date": "1579210032233",
-                    "requests": {
-                        "cache": {
-                            "enable": "false"
-                        }
-                    },
+                    "requests": {"cache": {"enable": "false"}},
                     "number_of_replicas": "2",
                     "uuid": "vOOsPNfxTJyQekkIo9TjPA",
-                    "version": {
-                        "created": "7050099"
-                    },
-                    "store": {
-                        "type": "fs"
-                    }
+                    "version": {"created": "7050099"},
+                    "store": {"type": "fs"},
                 }
-            }
+            },
         },
         # should be filtered
-        ".security": {
-            "mappings": {},
-            "settings": {
-                "index": {
-                    "number_of_shards": "1"
-                }
-            }
-        },
-        "geodata": {
-            "mappings": {},
-            "settings": {
-                "index": {
-                    "number_of_shards": "1"
-                }
-            }
-        }
+        ".security": {"mappings": {}, "settings": {"index": {"number_of_shards": "1"}}},
+        "geodata": {"mappings": {}, "settings": {"index": {"number_of_shards": "1"}}},
     }
     expected = {
         "osmgeopoints": {
-            "mappings": {
-                "dynamic": "strict",
-                "properties": {
-                    "location": {
-                        "type": "geo_point"
-                    }
-                }
-            },
+            "mappings": {"dynamic": "strict", "properties": {"location": {"type": "geo_point"}}},
             "settings": {
                 "index": {
                     "number_of_replicas": "{{number_of_replicas | default(2)}}",
                     "number_of_shards": "{{number_of_shards | default(3)}}",
-                    "requests": {
-                        "cache": {
-                            "enable": "false"
-                        }
-                    }
+                    "requests": {"cache": {"enable": "false"}},
                 }
-            }
+            },
         },
-        "geodata": {
-            "mappings": {},
-            "settings": {
-                "index": {
-                    "number_of_shards": "{{number_of_shards | default(1)}}"
-                }
-            }
-        }
+        "geodata": {"mappings": {}, "settings": {"index": {"number_of_shards": "{{number_of_shards | default(1)}}"}}},
```

Comment:
With the large line length, we have many examples like this across the codebase that are arguably more difficult to read, I can again try to track them down and add trailing commas. This can probably be done semi-automatically by looking at the git diff for "multiple lines that got collapsed to a single one".